dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/condition/PathQueryCondition.kt

```diff
@@ -12,6 +12,7 @@ import okhttp3.HttpUrl
  */
 data class PathQueryCondition(
     internal val path: String,
+    override val httpMethod: HTTPMethod,
```

Comment:
Scoring should also take into account that `httpMethod = ANY` is more general than any other value, shouldn't it?

Reply:
Sure! I've fixed the `compareTo` so `HTTPMethod.ANY` takes precedence over the other methods.
Nice catch! 👁

Let me know how do you see it now.

Reply:
I don't see any new commits.

Reply:
I forgot to push them! Sorry 🙂